Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.XamlHost/Microsoft.Toolkit.Forms.UI.XamlHost.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Forms.UI.XamlHost</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Forms.UI.XamlHost</AssemblyName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+
+    <Title>Windows Community Toolkit XAMLHost for Windows Forms</Title>
+    <Description>This library provides XAML islands interop helpers for Windows Forms. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>XAML Islands Win32 Forms WindowsForms WinForms XamlHost</PackageTags>
+  </PropertyGroup>
```

Comment:
This needs a different `TargetPlatformMinVersion` than specified for Win32 in props file. That version specifies RS4, which is the minimum required for WebView; however, the changes for Xaml Islands were not included until RS5. @GraniteStateHacker Do you recall what the minimum windows SDK was that included the WinMD changes for XAML islands?

Reply:
It was 17692. @ryalanms I'll update the CSPROJ with the appropriate minimum version

Reply:
Not sure but there may have been changes since 692